upcloud/resource_upcloud_server_test.go

```diff
@@ -456,6 +457,7 @@ func testAccServerNetworkInterfaceConfig(nis ...networkInterface) string {
 	`)
 
 	for i, ni := range nis {
+		netID := rand.Intn(255)
```

Comment:
This seems like a recipe for flaky tets :|

Reply:
this is only called by one test so, it's random only for one kind of test.